src/components/Well.vue

```diff
@@ -94,4 +136,9 @@ export default {
   .filled {
     fill: purple;
   }
+  // need to sort to add blur
```

Comment:
What do you mean here?

Reply:
It was sorted. Just forgot to remove the comment.